configuration/exodus-pipeline.yaml

```diff
@@ -438,10 +477,113 @@ Resources:
       TimeoutInMinutes: 10
     DependsOn: CodeBuildRole
 
+  IntegrationTestsCodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Condition: IsDev
+    Properties:
+      Name: !Sub ${project}-integration-tests
+      ServiceRole: !GetAtt CodeBuildRole.Arn
+      Environment:
+        Type: LINUX_CONTAINER
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
+      Source:
+        Type: CODEPIPELINE
+        BuildSpec: |
+          version: 0.2
+          phases:
+            install:
+              runtime-versions:
+                python: 3.7
+              commands:
+                - pip install --upgrade pip
+                - pip install tox
+            build:
+              commands:
+                - git clone https://github.com/$REPO_OWNER/$REPO_NAME.git -b $REPO_BRANCH
```

Comment:
The branch content could have changed already after the pipeline started and before we get to here.  Should it use a specific commit instead?

Reply:
Ah, yes, I missed this yesterday.

The `$REPO_BRANCH` variable will be "None" in most cases so that the default master/deploy branches are used, so this won't work. Instead of targeting a branch, I think we'll want to checkout #{SourceVars.CommitId}, which is the specific commit verified and deployed for each pipeline.

Note that the SourceVars namespace is only available in dev and stage pipelines, so this CodeBuild project will need to be created in one of those stacks.

Reply:
I would rather include the git repo as a secondary source in the prod pipeline.